src/generateZeroFunction.js

```diff
@@ -11,8 +11,12 @@
  * @return {function}
  *
  */
-function generateZeroFunction() {
-  // write code here
+function generateZeroFunction(getZero) {
+  return getZero;
 }
 
-module.exports = generateZeroFunction;
+const getZero = () => {
+  return 0;
```

Comment:
braces and return statement are optional. You can just do: 
`const getZero = () => 0;`